01_index.py

```diff
@@ -17,9 +18,14 @@
     environment = Environment.from_env_or_keyvault()
     config = Config(environment, args.config_path, args.data_dir)
 
+    # Are we running locally and not in AML? We do not want to run sampling on the distributed compute at this stage
+    local = False
```

Comment:
Shall we possibly rewrite it simpler?

`local = "01_index.py" in str(sys.argv[0])` 

Also maybe `is_local` to indicate it's bool type from the parameter name.

Reply:
Implemented